test/mocks/MockConfig.scala

```diff
@@ -46,4 +47,10 @@ class MockConfig(implicit val runModeConfiguration: Configuration) extends AppCo
   override val govUkGuidanceMtdVat: String = "mtd-vat"
   override val govUkGuidanceAgentServices: String = "agent-services"
   override val manageVatUrl: String = "/manage-vat"
+  override val routeToSwitchLanguage: String => Call = (lang: String) => controllers.routes.HelloWorldController.helloWorld()
```

Comment:
It does not seem this is being used in any tests however in manage-vat this line is different  

override val routeToSwitchLanguage: String => Call = (lang: String) => controllers.routes.LanguageController.switchToLanguage(lang)

Reply:
Done